### PR TITLE
PE-1449: calculate ranks

### PR DESCRIPTION
**Pull request by matibat**

### CODE REVIEW

**Review by fedellen — DISMISSED** [review]

File: src/queries.ts [hunk 1]
```diff
@@ -149,7 +146,8 @@ function createQuery(minBlock: number, maxBlock: number): Query {
 							values: [${VALID_APP_NAMES.map((appName) => `"${appName}"`)}]
```
Comment (fedellen) [inline src/queries.ts:146]: Were we going to query per app or continue querying all app names in one go? I recall some gateway DB issues with queries like this, but they may be patched up right now

**Review by fedellen — APPROVED** [review]

File: src/community/verto_contract_oracle.ts [hunk 2]
```diff
@@ -9,8 +9,8 @@ import { ContractReader } from './contract_oracle';
 export class VertoContractReader implements ContractReader {
 	/** Fetches smartweave contracts from the Verto cache */
 	public async readContract(txId: TransactionID): Promise<unknown> {
-		const response: AxiosResponse = await axios.get(`https://v2.cache.verto.exchange/${txId}`);
-		const contract = response.data;
+		const response = await fetch(`https://v2.cache.verto.exchange/${txId}`);
+		const contract = await response.json();
```
Comment (fedellen) [inline src/community/verto_contract_oracle.ts:13]: Not sure what you ran into with axios here, but it doesn't really matter. I only recommend axios because thats what arweave-js already bundles with

File: src/daily_output.ts [hunk 3]
```diff
@@ -61,8 +61,9 @@ export class DailyOutput {
 	 * @param {GQLEdgeInterface[]} queryResult the edges of ArFSTransactions only - 50 block before the latest
 	 */
 	public async feedGQLData(queryResult: GQLEdgeInterface[]): Promise<void> {
-		const aggregationPromises = queryResult.map(this.aggregateData);
-		await Promise.all(aggregationPromises);
+		for (const edge of queryResult) {
+			await this.aggregateData(edge);
+		}
```
Comment (fedellen) [inline src/daily_output.ts:66]: Why the change from concurrent promises?

File: src/community/verto_contract_oracle.ts (hunk 2 above)
Comment (karlprieb) [reply src/community/verto_contract_oracle.ts:13]: Yeap... We also don't know why it was not working propertly with Axios. It should be, but as we didn't want to spend more time on that we just switched back to node-fetch

File: src/daily_output.ts (hunk 3 above)
Comment (karlprieb) [reply src/daily_output.ts:66]: Probably a leftover from debugging

**Review by karlprieb — APPROVED** [review]

File: src/daily_output.ts (hunk 3 above)
Comment (matibat) [reply src/daily_output.ts:66]: We were spamming network requests to redstone. I think that's the reason why it took longer, because by doing it sequentially we got better timings
